sensor_msgs/Image[] images
string dataset
float32 confidence
float32 nms
string[] filter
string task
---
lasr_perception_server/Detection[] detected_objects
